src/pages/groupComparison/comparisonGroupManager/GroupCheckbox.tsx

```diff
@@ -74,6 +76,18 @@ export default class GroupCheckbox extends React.Component<IGroupCheckboxProps,
         return `${this.props.group.name} (${caseCounts(getNumSamples(this.props.group), getNumPatients(this.props.group))})`;
     }
 
+    @computed get editSubmitError() {
+        if (this.nameInput.length === 0) {
+            return { message: "Please enter a name."};
+        } else if (this.nameInput === this.props.group.name) {
+            return {};
```

Comment:
what does this empty object means here?

Reply:
It's a little confusing... it's to signify that there's an error, so that submit is disabled, but there's no message, so that there's no tooltip. I'll add a comment to explain that